jenni

```diff
@@ -20,6 +20,11 @@ from textwrap import dedent as trim
 dotdir = os.path.expanduser('~/.jenni')
 configpath = os.path.expanduser(dotdir + '/default.py')
 
+if os.geteuid() == 0:
```

Comment:
I think this is fine, but as a warning Linux doesn't actually require the root user have UID 0. It is *almost always correct* to assume this though.